flink/v1.14/flink/src/main/java/org/apache/iceberg/flink/source/FlinkSource.java

```diff
@@ -195,6 +211,7 @@ public FlinkInputFormat buildFormat() {
       } else {
         contextBuilder.project(FlinkSchemaUtil.convert(icebergSchema, projectedSchema));
       }
+      contextBuilder.exposeLocality(localityEnabled());
```

Comment:
Where is the `exposeLocality` variable used?

Reply:
https://github.com/apache/iceberg/blob/52808f5740bfac4943da5be63ed3380aa865d981/flink/v1.14/flink/src/main/java/org/apache/iceberg/flink/source/FlinkSource.java#L269-L272
It is used here to determine whether to use flink config.